tests/test_urlpatterns.py

```diff
@@ -164,18 +163,18 @@ def test_included_urls_django2_mixed(self):
             path('path', dummy_view)
         ]
         urlpatterns = [
-            url('^test/', include(nested_patterns), {'foo': 'bar'}),
+            path('test/', include(nested_patterns), {'foo': 'bar'}),
         ]
         self._test_included_urls(urlpatterns)
 
     @unittest.skipUnless(path, 'needs Django 2')
     def test_included_urls_django2_mixed_args(self):
         nested_patterns = [
             path('path/<int:child>', dummy_view),
-            url('^url/(?P<child>[0-9]+)$', dummy_view)
+            path('url/<int:child>', dummy_view)
         ]
         urlpatterns = [
-            url('^purl/(?P<parent>[0-9]+)/', include(nested_patterns), {'foo': 'bar'}),
+            path('purl/<int:parent>/', include(nested_patterns), {'foo': 'bar'}),
```

Comment:
@auvipy not sure if you had a chance to revert these changes, but this is the only thing blocking your PR from passing :slightly_smiling_face: .

There is a reason why there are two URL patterns in each of the lists, and they should not be the same thing 2x. The `url` one is testing that it leaves the arguments as strings, and the `path` one should parse integers and why the test is failing.

Reply:
gonna revert this soon